app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditViewModel.kt

```diff
@@ -14,18 +13,18 @@ import org.oppia.android.util.data.DataProviders.Companion.toLiveData
 import org.oppia.android.util.logging.ConsoleLogger
 import javax.inject.Inject
 
-// TODO(#1633): Fix ViewModel to not depend on View
 /** The ViewModel for [ProfileEditActivity]. */
 @ActivityScope
 class ProfileEditViewModel @Inject constructor(
-  private val activity: AppCompatActivity,
   private val logger: ConsoleLogger,
   private val profileManagementController: ProfileManagementController
 ) : ObservableViewModel() {
   private lateinit var profileId: ProfileId
-  private lateinit var switch: Switch
 
-  lateinit var profileName: String
+  private val _isAllowedDownloadAccess = MutableLiveData<Boolean>()
```

Comment:
This seems like a convention that we haven't been following in the code.
1. Has any reviewer suggested this change? If so, please mention the comment link.
2. If no one has suggested any comment related to this, I suggest you renaming to something else like
`isAllowedDownloadAccessLiveData`.

Reply:
No one has suggested this convention. Renamed to `isAllowedDownloadAccessMutableLiveData` as there is already an another livedata variable with same name.